### ISSUE TITLE
(Windows) Nativefier download file in app web

### ISSUE THREAD

**Issue by arres2k** [jiahaog/nativefier#370]
### Description
Problem download file in app web created by nativefier

I created an app (site joomla) with nativefier but the function download pdf or xls does not work when we click to download the file, while with chrome it works

### Specifications
- Version of Nativefier (run `$ nativefier --version`): `v0.0.0`
- Version of Node.js (run `$ node --version`): `v0.0.0`
- OS: `windows`
- Architecture `ai32`
- Stack trace from the error message (if any)

```
<Stack trace here>
```

**Comment by ronjouch** [comment]
Hi @arres2k, thanks for the feedback 🙂. I just tested Nativefier 7.3.1 and the just-released 7.4.0 under Windows, downloading works. The problem is that download status, completion, and path to downloaded files are completely invisible. We are aware of the problem, it's file as https://github.com/jiahaog/nativefier/issues/335 .

Closing the issue. Feel free to chime in again if I'm missing anything or with more info (website, command used to build nativefied app) if you're certain there's an actual problem with downloading.
